tsdb/config_test.go

```diff
@@ -9,12 +9,40 @@ import (
 
 func TestConfig_Parse(t *testing.T) {
 	// Parse configuration.
-	var c tsdb.Config
+	c := tsdb.NewConfig()
 	if _, err := toml.Decode(`
-enabled = false
+dir = "/var/lib/influxdb/data"
+wal-dir = "/var/lib/influxdb/wal"
 `, &c); err != nil {
 		t.Fatal(err)
 	}
 
-	// TODO: add remaining config tests
+	if err := c.Validate(); err != nil {
+		t.Errorf("unexpected validate error: %s", err)
+	}
+
+	if c.Dir != "/var/lib/influxdb/data" {
```

Comment:
nit: You can avoid having to write out the same expected literal twice by doing:

```
if got, exp := c.Dir, "/var/lib/influxdb/data"; got != exp {
        t.Errorf("unexpected dir:\n\ngot=%v\n\nexp=%v\n\n", got, exp)
}
```